tests/e2e/linalg/BUILD.bazel

```diff
@@ -92,11 +92,6 @@ iree_check_single_backend_test_suite(
         ],
     driver = "cuda",
     tags = [
-        # CUDA cuInit fails with sanitizer on.
-        "noasan",
-        "nomsan",
-        "notsan",
-        "noubsan",
```

Comment:
Hmmm... https://github.com/openxla/iree/pull/14157 only changes CMake, right? I suspect we'll still want to exclude CUDA tests from sanitizer configs with Bazel/Blaze... (we don't have sanitizer Bazel builds on our GitHub CI, but we do have a downstream Blaze build that uses the `noasan` tag)

(Sorry, the simplification is nice... but this might be a case where explicit is better than implicit)

Reply:
Actually (chatting with @GMNGeoffrey), maybe we can filter there on `--driver=cuda`, `requires-gpu-nvidia`, or `requires-gpu-sm80` as needed.